aroquev/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -59,20 +62,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    JsonObject jsonObj = new Gson().fromJson(request.getReader(), JsonObject.class);
+    String comment = jsonObj.get("commentBody").getAsString();
+
+    UserService userService = UserServiceFactory.getUserService();
+    String username = userService.getCurrentUser().getEmail();
```

Comment:
I think most people would not want their email to be used as their username.

Reply:
Yep, I plan on implementing the 'nickname' feature next to finish this login functionality.